go/tasks/pluginmachinery/flytek8s/config/config.go

```diff
@@ -35,7 +35,7 @@ type K8sPluginConfig struct {
 	// Provide additional environment variable pairs that plugin authors will provide to containers
 	DefaultEnvVars map[string]string `json:"default-env-vars" pflag:"-,Additional environment variable that should be injected into every resource"`
 	// Provide additional environment variable pairs whose values resolve from the plugin's execution environment.
-	DefaultEnvVarsFromEnv map[string]string `json:"default-env-vars-from-env" pflag:",Additional environment variable that should be injected into every resource"`
+	DefaultEnvVarsFromEnv map[string]string `json:"default-env-vars-from-env" pflag:"-,Additional environment variable that should be injected into every resource"`
```

Comment:
what does this do?

Reply:
The pflags tool is not able to generate command for `map`s. Without the `-` the `make generate` would fail. This is to disable it

Reply:
Let me know if it doesn't make sense.

Reply:
thank you for fixing!